### PR TITLE
docs: update SQL Editor docs and screenshots

### PR DESCRIPTION
**Pull request by LiuJi-Jim**
### Changes

- Update docs for new keyboard shortcuts in admin mode.
- Update screenshots for the schema panel and **Clear screen** button.

These features are to be released with Bytebase 1.12.0 (20230202)

### Screenshots

10 1 1 234_64543_docs_sql-editor_admin-mode(1600_900) (2)

### CODE REVIEW

**Review by tianzhou — COMMENTED** [review]
`Clear history` and `Clear screen` are 2 different things.

Looks like the button did both. And I feel we only need "Clear screen".

**Comment by LiuJi-Jim** [comment]
> `Clear history` and `Clear screen` are 2 different things.
> 
> Looks like the button did both. And I feel we only need "Clear screen".

Sounds reasonable. I think I need more work on this feature.

**Comment by tianzhou** [comment]
> > `Clear history` and `Clear screen` are 2 different things.
> > Looks like the button did both. And I feel we only need "Clear screen".
> 
> Sounds reasonable. I think I need more work on this feature to split history queries and history statements.

Sure. As mentioned before, I feel only offering "Clear screen" is good enough.

**Comment by tianzhou** [comment]
> > `Clear history` and `Clear screen` are 2 different things.
> > Looks like the button did both. And I feel we only need "Clear screen".
> 
> Sounds reasonable. I think I need more work on this feature to split history queries and history statements.

Sure. As mentioned before, I feel only offering "Clear screen" is good enough.

**Comment by LiuJi-Jim** [comment]
update. PTAL

File: content/docs/sql-editor/admin-mode.md [hunk 1]
```diff
@@ -17,3 +17,5 @@ Click **Admin Mode** button to open a separate Admin Mode tab. Note that this is
 In Admin Mode, you are not limited to run SELECT, DDL and DML change queries, but also allowed to run database administration SQL statements such as SHOW, SET, GRANT, etc.
 
 ![Admin Mode](/static/docs/sql-editor/admin-mode.webp)
+
+In Admin Mode, you can press <kbd>⌘ + Up / Down</kbd> or <kbd>Ctrl + Up / Down</kbd> to explore history statements backward or forward. Click the **Clear screen** button or use its keyboard shortcut <kbd>⇧ + ⌥ + C</kbd> to clear history queries on the screen.
```
Comment (tianzhou) [inline content/docs/sql-editor/admin-mode.md:21]:
```suggestion
In Admin Mode, click the **Clear screen** button or use its keyboard shortcut <kbd>⇧ + ⌥ + C</kbd> to clear the screen. You can also press <kbd>⌘ + Up / Down</kbd> or <kbd>Ctrl + Up / Down</kbd> to explore history statements backward or forward. 
```

**Review by tianzhou — APPROVED** [review]
Tweak the order to avoid confusing that `Clear screen` would affect statement history.

**Comment by LiuJi-Jim** [comment]
@ecmadao PTAL the plan descriptions and feature matrix.

**Review by ecmadao — APPROVED** [review]

**Pull request merged**